src/System.Security.Cryptography.Hashing/src/System/Security/Cryptography/KeyedHashAlgorithm.cs

```diff
@@ -0,0 +1,45 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Diagnostics;
+
+namespace System.Security.Cryptography
+{
+    public abstract class KeyedHashAlgorithm : HashAlgorithm
+    {
+        protected KeyedHashAlgorithm()
+        {
+        }
+
+        public virtual byte[] Key
+        {
+            get
+            {
+                return _key.CloneByteArray();
+            }
+
+            set
+            {
+                _key = value.CloneByteArray();
```

Comment:
Desktop had some code to throw if you tried to change the key after hashing began.  Is this something the subclasses need to do now?  Do we not have the code because the checking was only done in the Transform methods which we don't have here?